test/integration/simple-deployment-recursive-task/00-install.yaml

```diff
@@ -66,7 +72,3 @@ spec:
         - image: busybox
           name: busybox
           command: ['sh', '-c', 'echo The app is running! && sleep infinity']
-      initContainers:
```

Comment:
those sleeps were irrelevant in many cases and did nothing apart from artificially slowing the tests down, so I removed them where applicable